src/utils/string/index.ts

```diff
@@ -26,3 +26,7 @@ export const omitTypename = (object) =>
   JSON.parse(JSON.stringify(object), (key, value) =>
     key === "__typename" ? undefined : value
   );
+
+export interface selectedStrings {
```

Comment:
I'm not sure that this is the best place for this to live. It might be better suited living in the same file as usePatchStatusSelect hook.

Reply:
Done. I didn't think it belonged in the hook either but maybe there isn't a best place for it